src/openvic-simulation/ThreadPool.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <condition_variable>
+#include <cstdint>
+#include <mutex>
+#include <thread>
+#include <vector>
+
+#include "openvic-simulation/pop/PopValuesFromProvince.hpp"
+#include "openvic-simulation/types/Date.hpp"
+
+namespace OpenVic {
+	struct GoodInstance;
+	struct PopsDefines;
+	struct Strata;
+
+	struct ThreadPool {	
```

Comment:
```suggestion
	struct ThreadPool {
```
Stray tab at the end of the line, there are a couple more + some stray spaces across this file and `ThreadPool.cpp`, could you search and remove them